dapp/src/components/Collateral.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react'
+import { Typography } from '@originprotocol/origin-storybook'
+import { assetRootPath } from 'utils/image'
+import { PieChart } from 'react-minimal-pie-chart'
+import { formatCurrency } from '../utils/math'
+import useCollateralQuery from '../queries/useCollateralQuery'
+import { useStoreState } from 'pullstate'
+import ContractStore from 'stores/ContractStore'
+
+const Collateral = () => {
+  const collateral = useStoreState(ContractStore, (s) => {
```

Comment:
nit: Perhaps destructure the object here, rather than using `collateral.collateral` everywhere else?

Reply:
done